scripts/integrations/config.ini

```diff
@@ -0,0 +1,2 @@
+[DEFAULT]
+LICENSE = MIT
```

Comment:
should this be apache 2.0?

Reply:
Maybe?
I'm less familiar with opensource licenses and regulations,
I've placed it just for default value concept and will change it to whatever needed :)

Reply:
`Apache-2.0` for [compatibility with SPDX](https://spdx.org/licenses/), but I agree.